tests/examples/valid.yaml

```diff
@@ -1 +1,42 @@
 title: Drupal Accessibility Conformance Report
+name-product-version: Drupal 9.1
+product-description: Content Management System
+contact-information: mike.gifford@civicactions.com
+notes: Links to the issues identified are included where possible to ensure that this is a living document where outstanding issues are regularly reviewed for compliance. The Authoring tool is evaluated against ATAG 2.0, Part A and B. Incorporating feedback from the Drupal community.
+chapters:
+  - num: "1"
```

Comment:
I wonder about numbering vs some kind of string key for the chapters? I can see numbering differing between the different VPAT variants, and if they ever change it will be a pain to resync. If we use string keys (e.g. `hardware`) I think that would be remediate that.
We would still need the chapter numbers of course, to be able to order the output correctly, but these could be part of the catalog (since that would be specific to a certain variant and version of VPAT):
```
chapters:
  hardware:
    num: 4
    title: Hardware
```

Reply:
I think just like the name of the component and criteria numbers will likely come from the catalog as well. I think it will be more in the format:

```
chapters:
  - num: "4"
    title: "Hardware"
```

Reply:
What about the issue of chapter numbers differing between different VPAT editions/versions? This is why I was leaning towards a string key rather than a chapter number.

Reply:
Right, I was thinking that chapters in the current format would be a freeform array where the user decides the num and title. If we do a string key then we would have to define all the available keys. I can submit a PR to this branch to see how that would look.

Reply:
@grugnog submitted PR #58 to provide a string key solution.